Sources_v3/WebSocketClient/EventMiddlewares/TypingStartCleanupMiddleware.swift

```diff
@@ -4,15 +4,21 @@
 
 import Foundation
 
+extension TimeInterval {
+    /// The maximum time the incoming `typingStart` event is valid before a `typingStop` event is emitted automatically.
+    static let incomingTypingStartEventTimeout: TimeInterval = 30
+}
+
```

Comment:
This should come from `ChatClientConfig`

Reply:
I really don't think it's necessary. I think we should do the same as in other SDK's and use default values.